### PR TITLE
E UI 3134 manage options

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3134

### Change description ###
- Moving actions to right
- Setting title to 'Manage cases' from 'Manage Cases'
- Ensuring tab selected is shown with bold white underline

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/app/components/hmcts-global-header/hmcts-global-header.component.html [hunk 1]
```diff
@@ -57,7 +57,9 @@
       <nav class="hmcts-primary-navigation transparent-background" aria-label="Primary navigation">
         <ul class="hmcts-primary-navigation__list">
           <li class="hmcts-primary-navigation__item" *ngFor="let item of items" >
-            <a class="hmcts-primary-navigation__link" (click)="onEmitSubMenu(item)" [attr.aria-current]="item.active ? true: null" [routerLink]="item.href">{{item.text}}</a>
+
```
Comment (udaydenduluri33) [inline src/app/components/hmcts-global-header/hmcts-global-header.component.html:60]: please remove the blank line

File: src/app/app.constants.ts [hunk 2]
```diff
@@ -70,7 +70,7 @@ const applicationUserThemes: Theme[] =  [
   },
   {
     roles: ['pui-case-manager'],
-    appTitle: {name: 'Manage Cases', url: '/'},
+    appTitle: {name: 'Manage cases', url: '/'},
```
Comment (Mo-Lala84) [inline src/app/app.constants.ts:73]: The application is called Manage case, so the url is https://manage-case.platform.hmcts.net so can we please change it to "Manage case"?

**Review by Mo-Lala84 — APPROVED** [review]
lgtm subject to pr build passing

File: src/app/components/hmcts-global-header/hmcts-global-header.component.html [hunk 3]
```diff
@@ -57,7 +57,9 @@
       <nav class="hmcts-primary-navigation transparent-background" aria-label="Primary navigation">
         <ul class="hmcts-primary-navigation__list">
           <li class="hmcts-primary-navigation__item" *ngFor="let item of items" >
-            <a class="hmcts-primary-navigation__link" (click)="onEmitSubMenu(item)" [attr.aria-current]="item.active ? true: null" [routerLink]="item.href">{{item.text}}</a>
+            
+            <a class="hmcts-primary-navigation__link" (click)="onEmitSubMenu(item)" [attr.aria-current]="item === selectedItem || setTab === item.text ? true: null" [routerLink]="item.href">{{item.text}}</a>
```
Comment (udaydenduluri33) [inline src/app/components/hmcts-global-header/hmcts-global-header.component.html:61]: (item === selectedItem || setTab === item.text) ? true: null

File: src/app/components/hmcts-global-header/hmcts-global-header.component.ts [hunk 4]
```diff
@@ -25,14 +26,49 @@ export class HmctsGlobalHeaderComponent {
...
+  public tab;
 
-  constructor(public nocStore: Store<fromNocStore.State>) { }
+  constructor(
+    public nocStore: Store<fromNocStore.State>,
+    private readonly router: Router) { }
+
+  public ngOnInit() {
+    // set the active tab via the url in the router
+    this.tab = this.setActiveTab();
```
Comment (udaydenduluri33) [inline src/app/components/hmcts-global-header/hmcts-global-header.component.ts:38]: `this.tab = this.setActiveTab(this.router.url);`

File: src/app/containers/app-header/app-header.component.ts [hunk 5]
```diff
@@ -208,6 +208,10 @@ export class AppHeaderComponent implements OnInit, OnDestroy {
       showFindCase,
     } = applicationTheme;
 
+    // current ability to decapatilise the 'Cases'
+    if (appTitle.name === 'Manage Cases') {
```
Comment (udaydenduluri33) [inline src/app/containers/app-header/app-header.component.ts:212]: You dont need this

File: src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts [hunk 6]
```diff
@@ -1,14 +1,16 @@
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
 import { provideMockStore } from '@ngrx/store/testing';
 
 import { HmctsGlobalHeaderComponent } from './hmcts-global-header.component';
 
-describe('HmctsGlobalHeaderComponent', () => {
+fdescribe('HmctsGlobalHeaderComponent', () => {
```
Comment (udaydenduluri33) [inline src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts:9]: please remove this.

File: test/accessibility/helpers/accessibilityAuditor.js [hunk 7]
```diff
@@ -1,91 +1,91 @@
 const aChecker = require("accessibility-checker");
-const {conf} = require('../config/config');
+const { conf } = require('../config/config');
```
Comment (udaydenduluri33) [inline test/accessibility/helpers/accessibilityAuditor.js:2]: please revert the changes like this as they are not intentional [I think]

File: src/app/components/hmcts-global-header/hmcts-global-header.component.html (hunk 3 above)
Comment (connorpgpmcelroy) [reply src/app/components/hmcts-global-header/hmcts-global-header.component.html]: Edited this - no longer need selectedItem

**Review by udaydenduluri33 — APPROVED** [review]
LGTMc
